events/prioritizer.py

```diff
@@ -66,7 +71,8 @@ def get_prioritized(queryset, request):
                 if body in followed_bodies:
                     body_bonus += int(BODY_FOLLOWING_BONUS + (TIME_DEP_BODY_BONUS * start_time_factor))
             event.weight += body_bonus
-
+    #Apply Length Penalty
+    event.weight *= length_penalty
```

Comment:
This needs more indent, `event` isn't defined outside the loop. Check failing test cases. Also add a whitespace between `#` and the comment.